cmd/pkg.go

```diff
@@ -0,0 +1,119 @@
+// Copyright 2024 Commonwealth Scientific and Industrial Research Organisation (CSIRO) ABN 41 687 119 230
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"context"
+	"fmt"
+
+	sdk "github.com/ivcap-works/ivcap-cli/pkg"
+	"github.com/spf13/cobra"
+)
+
+var forcePush bool
+
+func init() {
+	rootCmd.AddCommand(pkgCmd)
+
+	pkgCmd.AddCommand(listPackageCmd)
+	pkgCmd.AddCommand(pushPackageCmd)
+	pushPackageCmd.Flags().BoolVarP(&forcePush, "force", "f", false, "Push packages even it already exists")
+	pkgCmd.AddCommand(pullPackageCmd)
+	pkgCmd.AddCommand(removePackageCmd)
+}
+
+var (
+	pkgCmd = &cobra.Command{
+		Use:     "pkg",
+		Aliases: []string{"pkgs", "package", "packages"},
```

Comment:
as mention in the core PR, I would prefer `packages` over `pkg`. I can see it is covered by the aliases, but can we reverse it?